docs/usermanual/userman_admin.rst

```diff
@@ -661,37 +670,45 @@ Available settings
 Most common settings
 ^^^^^^^^^^^^^^^^^^^^
 
-* DEBUG = <True or False> (default = False) -- Enables debug mode. In case you run into technical issues, enable this setting before troubleshooting / reporting.
-* CONTENT_ROOT = "<path to desired content folder>" (default=ka-lite/content)
-  This is the path that KA Lite will use to look for KA Lite video files to play.  Change the path to another local directory to get video files from that directory. NB! Directory has to be writable for the user running the server in order to download videos.
-* TIME_ZONE = <desired time zone>  (default = "America/Los_Angeles")
+* ``DEBUG = <True or False> (default = False)``
```

Comment:
It isn't very important, no. It doesn't change much to our LOGGING, but it would enable static serving when using kalite manage runserver.

But putting it under "Most common settings" is maybe a tad too much. Could you address this in #4106 ?